garrysmod/lua/includes/extensions/net.lua

```diff
@@ -17,6 +19,22 @@ function net.Receive( name, func )
 
 end
 
+-- WORKAROUND: net.BytesLeft is offset by an unknown amount, but relative offset is fine
+local bitsLeftStart, messgeLen = 0, 0
+net.BytesLeftInternal = net.BytesLeftInternal or net.BytesLeft
+
+--
+-- Returns the amount of data left to read in the current message. 
+-- Does nothing when sending data.
+--
+function net.BytesLeft()
+	local bytesLeft,bitsLeft = net.BytesLeft()
```

Comment:
Oversight here, this will cause an infinite recursion. Should use BytesLeftInternal